boards/arm/nrf9131ek_nrf9131/nrf9131ek_nrf9131.yaml

```diff
@@ -0,0 +1,17 @@
+identifier: nrf9131ek_nrf9131
+name: nRF9131-EK-NRF9131
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+ram: 256
```

Comment:
This should match the size of the node chosen as `zephyr,sram`, which is `sram0_s` in this case, so it should be 88 kB.

Reply:
done